### PR TITLE
Simplify metrics test code.

### PR DESCRIPTION
**Pull request by zebrik**

### CODE REVIEW

**Comment by zebrik** [comment]
@honza-kasik please review

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceMetricsTestCase.java [hunk 1]
```diff
@@ -1,97 +0,0 @@
...
-import org.openqa.selenium.WebDriver;
-
-import static org.junit.Assert.assertEquals;
-
-/**
- * @author mkrajcov <mkrajcov@redhat.com>
- */
-@RunWith(Arquillian.class)
-@Category(Shared.class)
-public class DataSourceMetricsTestCase {
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceMetricsTestCase.java:29]: moved to another package

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceRuntimeTestCase.java [hunk 2]
```diff
@@ -1,64 +0,0 @@
...
-import org.openqa.selenium.WebDriver;
-
-import static org.jboss.hal.testsuite.finder.FinderNames.*;
-
-/**
- * @author Harald Pehl
- */
-@RunWith(Arquillian.class)
-@Category(Domain.class)
-public class DataSourceRuntimeTestCase {
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/DataSourceRuntimeTestCase.java:44]: Removed as useless.

**Review by honza-kasik — APPROVED** [review]
Nice idea! Thanks! I didn't find any issues!

**Pull request merged**